Polish/ubranie/kieszen/gwizdek/zbadajgwizdek/zagwizdaj/zagwizdaj.md

```diff
@@ -0,0 +1 @@
+Nagle pojawia się postać ze skrzydłami. Gwałtownym ruchem chwyta Cię za rękę i momentalnie przenosicie się na ogrąmną łąkę na której jest tylko jedno, samotne drzewo.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.